lib/chef/provider.rb

```diff
@@ -390,9 +390,6 @@ def action_#{name}
           end
         end
       end
-
-      require "chef/dsl/recipe"
-      include Chef::DSL::Recipe
```

Comment:
Also as I think about this, does this change the behavior on non-12.5-style (i.e. non-fused) providers using `use_inline_resources`?